dockerhub/README.md

```diff
@@ -0,0 +1,186 @@
+## Real-time Acceleration Platform for Integrated Data Science
+
+* * *
+
+## What is RAPIDS?
+
+Visit [rapids.ai](http://rapids.ai) for more information.
+
+The RAPIDS suite of software libraries gives you the freedom to execute end-to-end data science and analytics pipelines entirely on GPUs. It relies on NVIDIA® CUDA® primitives for low-level compute optimization, but exposes that GPU parallelism and high-bandwidth memory speed through user-friendly Python interfaces.
+
+* * *
+
+## Prerequisites
+
+*   GPU support
+    *   Compute Capability 6.0 (Pascal Architecture) or higher
+*   CUDA support
+    *   9.2 or newer
+*   OS support
+    *   Ubuntu 16.04 LTS (tested and confirmed working)
+*   Docker support
+    *   [Docker CE v18+](https://docs.docker.com/install/linux/docker-ce/ubuntu/) - _apt for Ubuntu 16.04 **doesn't** include v18+ by default_
+    *   [nvidia-docker v2+](https://github.com/nvidia/nvidia-docker/wiki/Installation-%28version-2.0%29)
+
+* * *
+
+## Available Tags
+
+* `cuda9.2_ubuntu1604` - CUDA 9.2, Ubuntu 16.04
+    *  Based off of `nvidia/cuda:9.2-devel-ubuntu16.04`.
+    *  All code projects compiled with `nvcc` version 9.2.
+    *  Python 3.5 and gcc5.
+
+* `cuda10.0_ubuntu1604` - CUDA 10.0, Ubuntu 16.04
+    *  Based off of `nvidia/cuda:10.0-devel-ubuntu16.04`.
+    *  All code projects compiled with `nvcc` version 10.0.
+    *  Python 3.5 and gcc5.
+
+* _Coming soon_ - more tags supporing CUDA 10 and Ubuntu 18.04
+
+* * *
+
+## Usage
+
+1.  First launch an interactive session:
+
+        docker run --runtime=nvidia \
+                --rm -it \
+                -p 8888:8888 \
+                -p 8787:8787 \
+                -p 8786:8786 \
+                rapidsai/rapidsai:ubuntu1604_cuda92_py35
+
```

Comment:
No code block?

Reply:
I just converted what you wrote on dockerhub from HTML to MD using a convertor.

Reply:
Hold on, this *is* a code block. Note the indentation? It should render as a code block.

Reply:
My mistake. Thanks.